src/dm-area/dm-area.tsx

```diff
@@ -518,6 +519,15 @@ const Content = ({
   );
   const [cropperNode, selectFile] = useTokenImageUpload();
 
+  const LoadedMapDiv = styled.div`
```

Comment:
U should declare this component outside the other component. Otherwise, it will be recreated on every re-render.